src/dsa/rpo_stark/stark/mod.rs

```diff
@@ -1,11 +1,13 @@
+use alloc::vec::Vec;
 use core::marker::PhantomData;
 
 use prover::RpoSignatureProver;
-use rand::{distributions::Standard, prelude::Distribution, thread_rng, RngCore, SeedableRng};
+use rand::{distributions::Standard, prelude::Distribution};
```

Comment:
Do we still need these two imports? (they are used in defining trait bounds on line 31 below, but it is not clear to me why that bound is needed).

Reply:
Thanks, these should not be needed now. Removed!